### PR TITLE
first pass of triggers is complete

### PR DESCRIPTION
**Pull request by harley-meya**

### CODE REVIEW

File: Triggers/meya.keyword.yaml.meta.yaml [hunk 1]
```diff
@@ -1,7 +1,16 @@
 ---
 docstring: |
     ## meya.keyword.yaml
-    *TODO*
+    Trigger the flow when the user's input exactly matches the keyword.
+
+    This intent can be handy when testing and debugging your bot. For example, you may have hidden flows that print information pertinent only to developers. You can restrict access by using the keyword intent type with a hard to guess keyword, like `t3St1ng`.
```
Comment (amanieismail) [inline Triggers/meya.keyword.yaml.meta.yaml]: *Calls this an intent (vs trigger)

File: Triggers/meya.api_ai.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+triggers:
+  - type: meya.api_ai
+    settings:
+        client_access_token: YOUR_CLIENT_ACCESS_TOKEN
+        language: "en"
+        intent: "weather"
```
Comment (amanieismail) [inline Triggers/meya.api_ai.yaml]: Why did you go with specific instances of the settings vs a variable?

**Comment by harley-meya** [comment]
Changed `setting` to `property`.
Added two regex examples.
Changed trigger snippets to `<VARIABLE_NAME>` format.

**Comment by harley-meya** [comment]
To have the pythex.org link open in a different tab, you have to use an HTML `<a href>` tag. Markdown is supposedly able to render HTML, but it was appearing as raw text.
If the HTML doesn't work, I'd prefer to remove the link altogether (just leave it as `pythex.org`). If we revert to the Markdown link formatting, I could see someone clicking on it and losing unsaved work in Bot Studio.

File: Triggers/meya.api_ai.yaml.meta.yaml [hunk 3]
```diff
@@ -2,8 +2,8 @@
 docstring: |
     ## meya.api_ai.yaml
```
Comment (amanieismail) [inline Triggers/meya.api_ai.yaml.meta.yaml]: Remove the .yaml from the heading. Not necessary

**Pull request merged**